components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Deploying Builder with Chef Automate"
+description = "use Chef Automate to deploy Builder"
+date = 2019-11-19T14:10:15-08:00
+weight = 20
+draft = false
+bref = ""
+toc = true
++++
+
+## Overview
+Enterprise customers may wish to set up an on-premises Habitat Builder depot to store
+Habitat packages for use by their own Habitat Studios and Supervisors. Use Chef Automate
+to deploy and manage an on-premises Habitat Builder depot.
+
+{{< warning >}}
+The current implementation of Automate-deployed Habitat Builder is an early milestone
+intended for use in proof-of-concept deployments. We do not consider it a fully-mature
+feature, and you should not either.
+{{< /warning >}}
+
+### System requirements
+<!-- Question: What do we tell them about disk space requirements for Builder artifact
```

Comment:
https://github.com/habitat-sh/on-prem-builder/blob/master/on-prem-docs/content/docs/getting-started.md#requirements

Reply:
We should sync up those recommendations with the preflight checks